extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/PrivilegesBuilder.scala

```diff
@@ -93,6 +101,22 @@ object PrivilegesBuilder {
       }
     }
 
+    def mergeProjectionWithIdentifier(
```

Comment:
private

Reply:
As method `mergeProjectionWithIdentifier`  inside method `buildQuery`, 'private' modifier is unnecessary and not allowed here.

Reply:
sorry, my bad